JSActionAnalog.m

```diff
@@ -23,14 +23,18 @@ - (id) initWithIndex: (int)newIndex {
 }
 
 -(id) findSubActionForValue: (IOHIDValueRef) value {
+    int raw = IOHIDValueGetIntegerValue(value);
+    double parsed = [self getRealValue: raw];
+    
     if ([[subActions objectAtIndex: 2] active]) {
+        if (fabs(parsed) < 0.3) {
```

Comment:
Hey @citelao thanks for the PR! Could you pull out this `0.3` into a constant or something?